lib/screenplay/config/permanent_config.ex

```diff
@@ -148,12 +148,17 @@ defmodule Screenplay.Config.PermanentConfig do
 
     new_published_screens_config = get_new_published_screens(published_config, screens_to_publish)
 
+    screenplay_screens_to_add =
+      screens_to_publish
+      |> Enum.into(%{})
```

Comment:
Is this needed? It seems like we're going from a list, into a map, and then immediately back into a list.

Reply:
It is not, thank you.